skimage/measure/_regionprops.py

```diff
@@ -1136,6 +1143,10 @@ def regionprops(label_image, intensity_image=None, cache=True,
         Coordinate list ``(row, col)``of the region scaled by ``spacing``.
     **coords** : (N, 2) ndarray
         Coordinate list ``(row, col)`` of the region.
+    **circularity** : float
+        Ratio of the circumference of the region to the circumference of a
+        circle with the same area. A value of 1.0 indicates a perfect circle.
+        Values lower than 1.0 indicate an extended perimeter.
```

Comment:
```suggestion
        Values lower than 1.0 indicate an extended perimeter.
        Calculated as ``4π * area / perimeter^2``.
```
ImageJ provides the formula, why shouldn't we as long as we don't point the docstring at a reference.